Dynatrace/dynatrace-operator
# Added 'build label propagation' e2e test

# Description

The test should ensure that build label propagation feature works properly. It consists of five steps:
1) build labels propagation is disabled
2) default build labels are propagated only
1) all custom build labels are propagated
1) existing environment variables are preserved
1) invalid propagation (empty value of build variables due to invalid references to build labels)

Additionally

test/e2e/buildlabels target has been added

## How can this be tested?
1. Create testdata/secrets/single-tenant.yaml (see test/testdata/secrets-samples).
2. Use make command to start the test.
```
make test/e2e/buildlabels
```
## Checklist
